### PR TITLE
Added social media share buttons to blog post

### PR DESCRIPTION
**Pull request by AliciadW**

### CODE REVIEW

**Comment by AliciadW** [comment]
@longhotsummer please review and let me know what you think. I've added the share buttons below the blog post. The other option I was playing with was to add it to the top, but then I thought that a user will probably want to share after reading... 🤷‍♀️

File: _layouts/blog-post.html [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
-      {{ content }}
-    </div>
+  <div class="col-lg-8 mx-auto">
+    <img class="img-fluid my-4" src="{{ site.baseurl }}/img/posts/{{ page.image }}" alt="{{ page.image }}">
+    <p class="lead">{{ page.intro}}</p>
+    {{ content }}
+  </div>
+  <div class="col-lg-8 mx-auto">
+    <span class="pr-3">Share</span>
+    <span class="pr-3"><a href="https://twitter.com/home?status=https%3A//laws.africa/{{ page.url }}"><img src="{{ site.baseurl }}/img/social/twitter.svg" /></a></span>
```
Comment (longhotsummer) [inline _layouts/blog-post.html]: For the links, page.url already starts with a /, so `laws.africa/` -> `laws.africa`, and `{{ site.baseurl }}` can be remove.

File: _layouts/blog-post.html [hunk 2]
```diff
@@ -7,11 +7,15 @@
...
-      <p class="lead">{{ page.intro}}</p>
-      {{ content }}
-    </div>
+  <div class="col-lg-8 mx-auto">
+    <img class="img-fluid my-4" src="{{ site.baseurl }}/img/posts/{{ page.image }}" alt="{{ page.image }}">
+    <p class="lead">{{ page.intro}}</p>
+    {{ content }}
+  </div>
+  <div class="col-lg-8 mx-auto">
+    <span class="pr-3">Share</span>
```
Comment (longhotsummer) [inline _layouts/blog-post.html]: `Share:`

File: img/social/whatsapp.svg [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
```
Comment (longhotsummer) [inline img/social/whatsapp.svg]: Hmm. :)

File: _layouts/blog-post.html (hunk 1 above)
Comment (AliciadW) [reply _layouts/blog-post.html]: Removed

File: _layouts/blog-post.html (hunk 2 above)
Comment (AliciadW) [reply _layouts/blog-post.html]: Added

File: img/social/whatsapp.svg (hunk 3 above)
Comment (AliciadW) [reply img/social/whatsapp.svg]: Removed the weirdness

**Comment by AliciadW** [comment]
@longhotsummer updated as per your comments :-)

**Pull request merged**